app/main.py

```diff
@@ -2,0 +1,44 @@
+class Animal:
+
+    alive = []
+
+    def __init__(self, name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+
+        if self.health > 0:
+            return Animal.alive.append(self)
+
+    def remove_animal_list(self) -> None:
+        if self in Animal.alive:
+            Animal.alive.remove(self)
+
+    def __repr__(self) -> str:
+        return f"{{Name: {self.name}, " \
+            f"Health: {self.health}, " \
+            f"Hidden: {self.hidden}}}"
+
+
+class Herbivore(Animal):
+
+    def hide(self) -> None:
+        if self.hidden is False:
+            self.hidden = True
+        else:
+            self.hidden = False
```

Comment:
You can simplify this by using `self.hidden = not self.hidden` to toggle the boolean value. This aligns with the checklist recommendation to change a `bool` value in one line.